<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortingClass</class>
 <widget class="QMainWindow" name="SortingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sorting</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_back">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>811</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Inp_box">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>711</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Sort">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Clear_box">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>711</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Out_box">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>711</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Input :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>260</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>340</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sorted :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarr">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Sorting.qrc"/>
 </resources>
 <connections/>
</ui>
